code/extract_transform.py

```diff
@@ -0,0 +1,33 @@
+import json
+import pandas as pd
+
+
+# def load_json(data):
+    # Load JSON
+#x=pd.DataFrame()
+def load():
+    with open("../json/top-rated-movies-02.json") as f:
+        data = json.load(f)
+        # print(data)
+        df = pd.DataFrame(data)
+    return df
+
+#print(load(x))
+# JSON to DataFrame
+def json_to_df():
+    dataframe = load()
+    print(dataframe)
+    #dataframe
+    # type conversion
+    dataframe['genres'] = dataframe['genres'].astype('str').apply(
```

Comment:
Why do we need this?

Reply:
added relevant comments to all transformations.